torchgeo/datasets/bigearthnet.py

```diff
@@ -572,3 +576,430 @@ def plot(
+        },
+        'metadata': {'files': {'metadata.parquet': '55687065e77b6d0b0f1ff604a6e7b49c'}},
+    }
+
+    dir_file_names: ClassVar[dict[str, str]] = {
+        's1': 'BigEarthNet-S1',
+        's2': 'BigEarthNet-S2',
+        'maps': 'Reference_Maps',
+        'metadata': 'metadata.parquet',
+    }
+
+    # https://collections.sentinel-hub.com/corine-land-cover/readme.html
+    # Table 1 of https://bigearth.net/static/documents/Description_BigEarthNet_v2.pdf
+    clc_colors: ClassVar[dict[str, str]] = {
+        'Urban fabric': '#e6004d',
+        'Industrial or commercial units': '#cc4df2',
+        'Arable land': '#ffffa8',
+        'Permanent crops': '#e68000',
+        'Pastures': '#e6e64d',
+        'Complex cultivation patterns': '#ffe64d',
+        'Land principally occupied by agriculture, with significant areas of natural vegetation': '#e6cc4d',
+        'Agro-forestry areas': '#f2cca6',
+        'Broad-leaved forest': '#80ff00',
+        'Coniferous forest': '#00a600',
+        'Mixed forest': '#4dff00',
+        'Natural grassland and sparsely vegetated areas': '#ccf24d',
+        'Moors, heathland and sclerophyllous vegetation': '#a6ff80',
+        'Transitional woodland, shrub': '#a6f200',
+        'Beaches, dunes, sands': '#e6e6e6',
+        'Inland wetlands': '#a6a6ff',
+        'Coastal wetlands': '#ccccff',
+        'Inland waters': '#80f2e6',
+        'Marine waters': '#e6f2ff',
+    }
+
+    clc_codes: ClassVar[dict[int, int]] = {
+        111: 0,  # Continuous Urban fabric
+        112: 0,  # Discontinuous Urban fabric
+        121: 1,  # Industrial or commercial units
+        211: 2,  # Non-irrigated arable land
+        212: 2,  # Permanently irrigated land
+        213: 2,  # Rice fields
+        221: 3,  # Vineyards
+        222: 3,  # Fruit trees and berry plantations
+        223: 3,  # Olive groves
+        231: 4,  # Pastures
+        241: 3,  # Annual crops with permanent crops
+        242: 5,  # Complex cultivation patterns
+        243: 6,  # Land principally occupied by agriculture...
+        244: 7,  # Agro-forestry areas
+        311: 8,  # Broad-leaved forest
+        312: 9,  # Coniferous forest
+        313: 10,  # Mixed forest
+        321: 11,  # Natural grassland
+        322: 12,  # Moors and heathland
+        323: 12,  # Sclerophyllous vegetation
+        324: 13,  # Transitional woodland/shrub
+        331: 14,  # Beaches, dunes, sands
+        333: 11,  # Sparsely vegetated areas
+        411: 15,  # Inland marshes
+        412: 15,  # Peatbogs
+        421: 16,  # Salt marshes
+        422: 16,  # Salines
+        511: 17,  # Water courses
+        512: 17,  # Water bodies
+        521: 18,  # Coastal lagoons
+        522: 18,  # Estuaries
+        523: 18,  # Sea and ocean
+    }
+
+    valid_splits = ('train', 'val', 'test')
+
+    def __init__(
+        self,
+        root: Path = 'data',
+        split: str = 'train',
+        bands: str = 'all',
```

Comment:
People have complained that some datasets have (B1, B2, etc.) and others have s1, s2, etc. I know you're just matching the existing dataset, but should we change both to the former?